projects/com.oracle.truffle.llvm.parser.base/src/com/oracle/truffle/llvm/parser/base/model/enums/Linkage.java

```diff
@@ -46,7 +46,8 @@
     AVAILABLE_EXTERNALLY,
     LINKERPRIVATE,
     LINKERPRIVATE_WEAK,
-    LINKONCEODRAUTOHIDE;
+    LINKONCEODRAUTOHIDE,
+    UNKNOWN; // TODO: required by LLVM IR Parser, should be removed when no longer needed
```

Comment:
The default linkage is 'external'. LLVM Documentation says it is implied if none is explicitly given, so there should never be unknown linkage. How can UNKNOWN happen in the Textparser?